### PR TITLE
FIX-TEMPLATE: 101-vm-from-sig

### PR DESCRIPTION
**Pull request by javid-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-vm-from-sig/prereqs/prereq.azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+            "max": "64"
+          },
+          "memory": {
+            "min": "2048",
+            "max": "307720"
+          }
+        }
+      }
+    },
+    {
```
Comment (bmoore-msft) [inline 101-vm-from-sig/prereqs/prereq.azuredeploy.json:126]: remove the unused resources (all but the sig resources)

File: 101-vm-from-sig/azuredeploy.json [hunk 2]
```diff
@@ -162,11 +150,6 @@
         "hardwareProfile": {
           "vmSize": "Standard_A1"
         },
-        "osProfile": {
```
Comment (bmoore-msft) [inline 101-vm-from-sig/azuredeploy.json:165]: revert this change (and the parameter changes with it)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-vm-from-sig/prereqs/prereq.azuredeploy.json (hunk 1 above)
Comment (bmoore-msft) [reply 101-vm-from-sig/prereqs/prereq.azuredeploy.json:126]: vnet, nic and vm

Comment (javid-spektra) [reply 101-vm-from-sig/prereqs/prereq.azuredeploy.json:126]: Gallery image version storageProfile property requires it. Is there any other way to provide a storage profile without creating VM?
